wire-ios-mocktransport/Tests/Source/Push Events/MockTransportSessionTeamEventsTests.swift

```diff
@@ -35,7 +35,7 @@ class MockTransportSessionTeamEventsTests: MockTransportSessionTests {
 
         XCTAssertEqual(payload["team"] as? String, teamIdentifier, "Wrong team identifier", file: file, line: line)
         guard let date = (payload as NSDictionary).optionalDate(forKey: "time") else { XCTFail("Event should have time", file: file, line: line); return }
-        XCTAssertLessThan(date, Date(), "Event date should be in the past", file: file, line: line)
+        XCTAssertLessThanOrEqual(date, Date(), "Event date should be in the past", file: file, line: line)
```

Comment:
This is flaky for me, sometimes the test seems to run so quick, that date and now are the same value.